dts/arm/nordic/nrf54l15_cpuapp_peripherals_ns.dtsi

```diff
@@ -0,0 +1,402 @@
+	status = "disabled";
+	reg = <0xce000 0x1000>;
+	cc-num = <6>;
+	max-bit-width = <32>;
+	interrupts = <206 NRF_DEFAULT_IRQ_PRIORITY>;
+	prescaler = <0>;
+};
+
+adc: adc@d5000 {
+	compatible = "nordic,nrf-saadc";
+	reg = <0xd5000 0x1000>;
+	interrupts = <213 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+	#io-channel-cells = <1>;
+};
+
+nfct: nfct@d6000 {
+	compatible = "nordic,nrf-nfct";
+	reg = <0xd6000 0x1000>;
+	interrupts = <214 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+temp: temp@d7000 {
+	compatible = "nordic,nrf-temp";
+	reg = <0xd7000 0x1000>;
+	interrupts = <215 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+gpio1: gpio@d8200 {
+	compatible = "nordic,nrf-gpio";
+	gpio-controller;
+	reg = <0xd8200 0x300>;
+	#gpio-cells = <2>;
+	ngpios = <16>;
+	status = "disabled";
+	port = <1>;
+	gpiote-instance = <&gpiote20>;
+};
+
+gpiote20: gpiote@da000 {
+	compatible = "nordic,nrf-gpiote";
+	reg = <0xda000 0x1000>;
+	interrupts = <219 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+	instance = <20>;
+};
+
+i2s20: i2s@dd000 {
+	compatible = "nordic,nrf-i2s";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	reg = <0xdd000 0x1000>;
+	interrupts = <221 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+qdec20: qdec@e0000 {
+	compatible = "nordic,nrf-qdec";
+	reg = <0xe0000 0x1000>;
+	interrupts = <224 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+qdec21: qdec@e1000 {
+	compatible = "nordic,nrf-qdec";
+	reg = <0xe1000 0x1000>;
+	interrupts = <225 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+grtc: grtc@e2000 {
+	compatible = "nordic,nrf-grtc";
+	reg = <0xe2000 0x1000>;
+	cc-num = <12>;
+	owned-channels = <0 1 2 3 4 5 6 7 8 9 10 11>;
+	interrupts = <228 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+dppic30: dppic@102000 {
+	compatible = "nordic,nrf-dppic";
+	reg = <0x102000 0x808>;
+	status = "disabled";
+};
+
+i2c30: i2c@104000 {
+	compatible = "nordic,nrf-twim";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	reg = <0x104000 0x1000>;
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+	interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+	easydma-maxcnt-bits = <16>;
+	status = "disabled";
+};
+
+spi30: spi@104000 {
+	/*
+	 * This spi node can be either SPIM or SPIS,
+	 * for the user to pick:
+	 * compatible = "nordic,nrf-spim" or
+	 *              "nordic,nrf-spis".
+	 */
+	compatible = "nordic,nrf-spim";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	reg = <0x104000 0x1000>;
+	interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+	max-frequency = <DT_FREQ_M(8)>;
+	easydma-maxcnt-bits = <16>;
+	status = "disabled";
+};
+
+uart30: uart@104000 {
+	compatible = "nordic,nrf-uarte";
+	reg = <0x104000 0x1000>;
+	interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+	status = "disabled";
+};
+
+gpio0: gpio@10a000 {
+	compatible = "nordic,nrf-gpio";
+	gpio-controller;
+	reg = <0x10a000 0x300>;
+	#gpio-cells = <2>;
+	ngpios = <5>;
+	status = "disabled";
+	port = <0>;
+	gpiote-instance = <&gpiote30>;
+};
+
```

Comment:
I see that both watchdog30 and 31 have been removed with the rationale that they are secure-only, but it is only WDT30 that is secure-only.

Reply:
You are right, I will add the 31 watchdog.